content/en/docs/started/introduction.md

```diff
@@ -4,11 +4,43 @@ description = "An introduction to Kubeflow"
 weight = 1
 +++
 
-The Kubeflow project is dedicated to making deployments of machine learning (ML)
-workflows on Kubernetes simple, portable and scalable. Our goal is not to
-recreate other services, but to provide a straightforward way to deploy
-best-of-breed open-source systems for ML to diverse infrastructures. Anywhere
-you are running Kubernetes, you should be able to run Kubeflow.
+## What is Kubeflow ?
+
+Kubeflow is a community and ecosystem of open-source projects to address each stage in the
+machine learning (ML) lifecycle. It makes ML on Kubernetes simple, portable, and scalable.
+The goal of Kubeflow is to facilitate the orchestration of Kubernetes ML workloads and to empower
+users to deploy best-in-class open-source systems to any Cloud infrastructure.
+Whether you’re a researcher, data scientist, ML engineer, or a team of developers, Kubeflow offers
+modular and scalable tools that cater to all aspects of the ML lifecycle: from building ML models to
+deploying them to production for AI applications.
+
+## What are Kubeflow Standalone Components?
+
+Kubeflow is composed of multiple, independent open-source projects which address different aspects
+of a ML lifecycle. These standalone components are designed to be usable both within the Kubeflow
+Platform and independently. These components can be installed independently on a Kubernetes cluster,
+providing flexibility to users who may not require the full capabilities of Kubeflow Platform but
+wish to leverage specific ML functionalities.
+
+## What is Kubeflow Platform ?
+
+The Kubeflow Platform refers to the full suite of Kubeflow components bundled together with
+additional integration and management tools. Installing Kubeflow as a platform means deploying a
+comprehensive ML toolkit that integrates these components into a cohesive system, optimized for
+managing the end-to-end ML lifecycle. These includes not only the standalone components but also:
```

Comment:
suggested change - This includes the standalone components coupled with these integrations and management tools: